### PR TITLE
Removing accessories from a jumpsuit is now a radial menu

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Removing accessories is now a radial menu

Adds a new radial helper, where all the icon work is done for you, and allows for multiple inputs of the same item.

## Why It's Good For The Game
Picking text from a list is boring, new using radial menus is fun and cool!

## Images of changes

image

## Testing

I discovered some bugs, fixed them, broke my breakpoints, fixed my breakpoints

## Changelog
:cl:
tweak: Removing accessories from a jumpsuit is now a radial menu
/:cl:

### CODE REVIEW

**Review by Burzah — APPROVED** [review]

File: code/_onclick/hud/radial.dm [hunk 1]
```diff
@@ -309,4 +309,45 @@ GLOBAL_LIST_EMPTY(radial_menus)
 	GLOB.radial_menus -= uniqueid
 	return answer
 
+/**
+ * Similar to show_radial_menu, but choices is a list of atoms, for which icons will be automatically generated.
+ * Supports multiple items of the same name, 2 soaps will become soap (1) and soap (2) to the user
+ */
+/proc/radial_menu_helper(mob/user, atom/anchor, list/choices, uniqueid, radius, datum/callback/custom_check, require_near = FALSE)
```
Comment (lewcc) [inline code/_onclick/hud/radial.dm:316]: Can you document the arguments for this proc in the standard format? Would help other people use this down the line

File: code/_onclick/hud/radial.dm [hunk 2]
```diff
@@ -309,4 +309,45 @@ GLOBAL_LIST_EMPTY(radial_menus)
...
+	for(var/atom/atom in choices)
+		if(!duplicate_amount.Find(atom.name))
+			duplicate_amount[atom.name] = 0
+		else
+			duplicate_amount[atom.name]++
+	var/list/duplicate_indexes = duplicate_amount.Copy()
+
+	var/list/icon_state_choices = list()
+	var/list/return_choices = list()
+	for(var/atom/atom in choices)
```
Comment (lewcc) [inline code/_onclick/hud/radial.dm:327]: I know it's not really a reserved keyword, but can you make use of a different variable name that isn't just `atom`?

File: code/_onclick/hud/radial.dm (hunk 1 above)
Comment (Contrabang) [reply code/_onclick/hud/radial.dm:316]: All of the arguments here are the same as for show_radial_menu except choices. I'm not exactly sure what all of them do in context to radial menus, and the original proc doesnt have context for them either.

Comment (lewcc) [reply code/_onclick/hud/radial.dm:317]: Fair enough

**Review by lewcc — APPROVED** [review]

**Pull request merged**